### PR TITLE
[BUGFIX] Versions weren't displayed correctly after #149

### PR DESCRIPTION
**Pull request by Madii**
-> [FEATURE] Icons for versions

### CODE REVIEW

**Comment by NamelessCoder** [comment]
No `@TODO`s in code please - it's **great** if you plan to implement the feature but until it is possible, just define the icon as an empty string and note in the commit message or pull request whether or not you plan to also implement the actual icon resolving ;)

**Comment by Madii** [comment]
ok done. Didn't know that **I** was planing to implement it :D

**Comment by Madii** [comment]
Interesting...didn't know my own commits will land here for check although no pull request.

**Comment by Madii** [comment]
I have no clue why he don't likes the one return line. Maybe some1 can help.

File: Classes/Service/ConfigurationService.php [hunk 1]
```diff
@@ -7,11 +7,13 @@
  * For the full copyright and license information, please read the
  * LICENSE.md file that was distributed with this source code.
  */
-
```
Comment (NamelessCoder) [inline Classes/Service/ConfigurationService.php]: Whitespace should be preserved.

File: Classes/UserFunction/ProviderField.php [hunk 2]
```diff
@@ -145,11 +146,13 @@ public function createVersionsField(array $parameters) {
...
-			$options = array();
-		}
-		return $this->renderSelectField($parameters, $options, $preSelectedVersion);
-	}
+			foreach ($versions as $version) {
+				$icon = $this->configurationService->getIconFromVersion($preSelectedVariant, $parameters['row']['CType'], $version);
+				$versionIcon = '<img src="' . $icon . '" alt="" /> ';
+				$options[$version] = array($versionIcon, $version);
+			}
+        }
```
Comment (NamelessCoder) [inline Classes/UserFunction/ProviderField.php]: This, and next two lines, use spaces for indentation - must be tabs.

**Comment by Madii** [comment]
@NamelessCoder This is still open :D

**Comment by NamelessCoder** [comment]
Please see comments in code - CGL issues need to be fixed (and since it is this old, you should also rebase this on master to make sure it is compatible).

**Pull request closed without merging**